### ISSUE TITLE
Setting Type Broken on Linux

### ISSUE THREAD

**Issue by IsaacElenbaas** [qutebrowser/qutebrowser#5021]
content.user_stylesheets can be an array of strings or a string. I switched to Arch yesterday, and on loading my config file I got 'expected a value of type list but got str' where I set content.user_stylesheets. I keep it as type string because when I run config-cycle later, if it's not a string I will have to cycle twice the first time (as the initial value doesn't match either of the config-cycle values).

**Comment by The-Compiler** [comment]
Works fine for me. What version are you on, how does your config.py look, and what error do you get exactly?

**Comment by IsaacElenbaas** [comment]
Pointing to my user_stylesheets line (I even commented it when changing, I'm sure that was it) with incorrect type as quoted above, and yet when I revert it now to a string instead of a list it's fine. No error on Windows, I transfer, and I got it, so I don't see why I can't get it to happen now, but nothing to do if I can't reproduce it myself lol. I'll see if it isn't in my logs somewhere. All that's changed since then is I installed fonts and steam.
